website/docs/cloud-docs/agents/agents.mdx

```diff
@@ -40,32 +43,29 @@ To customize this update behavior, pass the flag `-auto-update` or set the envir
 | `patch`        | The agent only updates to the newest patch version, new minor versions require a manual update. |
 | `disabled`     | Disables automatic updates, all updates are manual.
 
-## Start the Agent
+## Run an Agent Using the Official Docker Image
+
+### Start the Agent
```

Comment:
everything below this is the instructions for starting an agent using the binary i think. i think the docker section got accidentally removed but it does seem like it would look basically [the same as this but just with the docker commands instead](https://github.com/hashicorp/terraform-docs-agents/commit/9f57b1f679892bb18913abb018eb09413ce39933#diff-8246c2c007958b7b9c27e46d5b08d5ab9be5df177c25278c5008cb4c376e30fcR37)

Reply:
good catch